.env.test

```diff
@@ -20,3 +20,8 @@ BROWSER_TESTS_USER_NAME=
 BROWSER_TESTS_USER_PASSWORD=
 BROWSER_TESTS_JWT_SIGN_SECRET=
 VITE_HELSINKI_PROFILE_URL=
+VITE_MATOMO_URL_BASE="//webanalytics.digiaiiris.com/js/"
+VITE_MATOMO_SITE_ID=813
```

Comment:
I cannot verify whether this ID is a right one or not, so please double check it, so the stats are collected underright project.

Reply:
its in the ticket in the comment